src/styles/blocks/card.scss

```diff
@@ -0,0 +1,95 @@
+.product-card {
+  box-sizing: border-box;
+  padding-inline: 16px;
+  width: 200px;
+  background-color: $main-background-color;
+  display: flex;
+  flex-direction: column;
+  border-radius: 5px;
+  border: 1px solid rgba(243, 243, 243, 1);
+
+  &__image {
+    object-position: center;
+    margin-inline: 3px;
+    width: 160px;
+    height: 134px;
+    margin-top: 32px;
+  }
+
+  &__product-title {
+    margin-top: 40px;
+    margin-bottom: 0;
+    height: 36px;
+    font-size: 12px;
+    line-height: 18px;
+    text-align: left;
+    font-weight: 500;
+    color: $bold-font-color;
```

Comment:
The variable `$bold-font-color` is used here but it is not defined in this file. Ensure it is defined to prevent compilation errors.